tools/generate_gclient-xwalk.py

```diff
@@ -36,7 +36,7 @@ def GenerateGClientXWalk(options):
   with open(os.path.join(CROSSWALK_ROOT, 'DEPS.xwalk')) as deps_file:
     deps_contents = deps_file.read()
 
-  if 'XWALK_OS_ANDROID' in os.environ:
+  if os.environ.get('XWALK_OS_ANDROID', '') == '1':
```

Comment:
You don't need to pass the second parameter here, `None == '1'` is also a valid comparison.